sycl/include/sycl/ext/oneapi/bindless_images.hpp

```diff
@@ -683,7 +683,9 @@ get_image_num_channels(const image_mem_handle memHandle,
 namespace detail {
 
 // is sycl::vec
-template <typename T> struct is_vec { static constexpr bool value = false; };
+template <typename T> struct is_vec {
```

Comment:
I guess this was forced by clang-format and there was no real change here, right? If so, please, return back to original state so that blame can point to the last meaningful change.

Reply:
Yes, good point. Reverted.